pom.xml

```diff
@@ -108,6 +108,18 @@
       <version>2.28.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.debezium</groupId>
+      <artifactId>debezium-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+    </dependency>
```

Comment:
Let's add the right Quarkus extension instead. It will apply all the right config required for running the Kafka client in a native binary.